apps/main-app/src/constants.js

```diff
@@ -248,6 +248,9 @@ export const SPECIES = [
     apolloName: 'human',
     apolloTrack: '/All%20Genes/',
     jBrowseName: 'Homo sapiens',
+    jBrowsenclistbaseurl: 'https://s3.amazonaws.com/agrjbrowse/docker/5.4.0/human/',
+    jBrowseurltemplate: 'tracks/All_Genes/{refseq}/trackData.jsonz',
+    jBrowsefastaurl: 'https://s3.amazonaws.com/agrjbrowse/fasta/GCF_000001405.40_GRCh38.p14_genomic.fna.gz',
```

Comment:
One thing that we might want to do is put cloudfront in front of this S3 bucket... it will speed up the delivery of these files and it costs less.

Reply:
That's (I assume) a reasonable idea. I don't know anything about cloudfront, so I'll defer to you.

Reply:
In any event, it should impede testing.